ai/ide-extension.md

```
@@ -5,68 +5,141 @@ icon: puzzle-piece
-2. Click Set API Key and paste your Currents [API key](https://app.gitbook.com/s/lcxad7NaXT7D2V6owvHN/get-started/authentication#managing-the-api-keys).
-3. Select a project — your runs will appear immediately.
+2. Click **Set API Key** and paste your Currents [API key](https://app.gitbook.com/s/lcxad7NaXT7D2V6owvHN/get-started/authentication#managing-the-api-keys).
+3. Select a project -- your runs appear immediately.
 
-### Features
+<!-- TODO: screenshot — Currents panel in the Activity Bar showing the Set API Key prompt and project picker -->
 
-<figure><img src="../.gitbook/assets/image (69).png" alt=""><figcaption></figcaption></figure>
+## Features
 
 #### Run Feed
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)